DoSomethingArchive/legacy-website
# Makes taxonomy terms translatable

#### What's this PR do?
- Updates dosomething_taxonomy feature so it's not overridden
- Saves entity translation type settings
- Enables taxonomy translation
- Saves default taxonomy vocabularies translation settings
- Configures Taxonomy Vocabularies translation
- Moves up taxonomies language field to the top after Name field
- Replaces Taxonomy Name and Description with translatable fields
- Provides a tool to fix taxonomy term fields replaced by title module
- Provides `ds-global` routine to fix taxonomy translatable fields
- Enables translation of taxonomy's field_title
- Fixes taxonomy's field_title translations
#### How should this be manually tested?
- Download branch
- Import Prod database
- Run `ds-global make-taxonomy-translatable`
- Go to http://dev.dosomething.org:8888/us/taxonomy/term/1/edit, check all fields are present
- Click Save / View
- Translate term, check that translation is viewable
#### Any background context you want to provide?

@angaither @deadlybutter: Default language for taxonomies need to be set to 'English', because term entity doesn't have actual language column and it's language is determined only in Entity Translation tables. Should we alter the language on creation for regional admins?

@deadlybutter Do we need any specific redirects for taxonomies, as we made for nodes?

@mikefantini @sheyd  Taxonomy Translation Matrix: https://docs.google.com/spreadsheets/d/1nYdrVRh67dpRvr7CmGXNeKUWY0PkbqgnS3pPqDRzFqc/edit#gid=0&vpid=A2
#### What are the relevant tickets?
- Fixes #5758 
- Closes #5715
## 

CC @sheyd @mikefantini @deadlybutter
